tls-client/HelloHttpsClient.cpp

```diff
@@ -363,9 +366,18 @@ void HelloHttpsClient::sslDebug(void *ctx, int level, const char *file,
 int HelloHttpsClient::sslVerify(void *ctx, mbedtls_x509_crt *crt, int depth,
                                 uint32_t *flags)
 {
-    HelloHttpsClient *client = static_cast<HelloHttpsClient *>(ctx);
+    int ret = 0;
+
+    /*
+     * If MBEDTLS_HAVE_TIME_DATE is defined, then the certificate date and time
+     * validity checks will probably fail because this application does not set
+     * up the clock correctly. We filter out date and time related failures
+     * instead
+     */
+    *flags &= ~MBEDTLS_X509_BADCERT_FUTURE & ~MBEDTLS_X509_BADCERT_EXPIRED;
 
-    int ret = -1;
+#if HELLO_HTTPS_CLIENT_DEBUG_LEVEL > 0
```

Comment:
@mazimkhan / @andresag01 - Will changing what's displayed as debug impact the CI, which uses text output to validate the example?

Reply:
This does not change what is displayed on the terminal unless the HELLO_HTTPS_CLIENT_DEBUG_LEVEL macro is > 0 (which I think isnt by default). In fact, the  changes in the placing of the macro is to ensure this happens. Note that in the past the sslVerify function cas not configured unless HELLO_HTTPS_CLIENT_DEBUG_LEVEL> 0. However, now it is because we need it to clear the timing flags. Hence, I had to put extra macro checks around the printing code in sslVerify() to prevent it from printing by default.

cc @k-stachowiak: I think this answers your question as well.

Reply:
If this change does not modify the prints that appear in `mbed-os-example-tls/tests/tls-client.log` then this will not impact the CI. Any additional prints are ignored.